renovate.json

```diff
@@ -23,8 +23,8 @@
 	"ignoreUnstable": false,
 	"baseBranches": [
 		"main",
-		"stable4.4",
-		"stable4.3"
+		"stable4.7",
+		"stable4.6"
```

Comment:
Do we really need to support 4.6? It is only available on Nextcloud stable25 and AFAIK we support down to stable27.

Reply:
True, removing it